guacamole/src/main/webapp/app/form/directives/form.js

```diff
@@ -163,6 +181,59 @@ angular.module('form').directive('guacForm', [function form() {
 
             });
 
+            /**
+             * Returns whether the given field should be displayed to the
+             * current user.
+             *
+             * @param {Field} field
+             *     The field to check.
+             *
+             * @returns {Boolean}
+             *     true if the given field should be visible, false otherwise.
+             */
+            $scope.isVisible = function isVisible(field) {
+
+                // Forms with valuesOnly set should display only fields with
+                // associated values in the model object
+                if ($scope.valuesOnly)
```

Comment:
Yep. It's definitely not needed by these changes, and the changes which *did* need it were ultimately reverted.